lib/mayaUsd/nodes/proxyShapeUpdateManager.cpp

```diff
@@ -20,6 +20,8 @@
 #include <pxr/usd/usd/tokens.h>
 #include <pxr/usd/usdUI/tokens.h>
 
+#include <mutex>
```

Comment:
No sure why this file got modified, was this to fix a build issue?

Reply:
Yeah, I couldn't build locally without that change. It's PR'd here:
https://github.com/Autodesk/maya-usd/pull/2975